hack/tools/binfolder/controller-gen

```diff
@@ -0,0 +1 @@
+/home/swastik_gour/github/cluster-api-provider-azure/hack/tools/bin/controller-gen-v0.9.2
```

Comment:
I think these files under `/home` and the `minikube-linux-amd64` binary were mistakenly added to this commit.

Reply:
@mboersma please check for any further discrepancy